.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
     types: [created]
 
+env:
+  FORMULA: cbmc-viewer
+  TAP: aws/tap
+  BOT_USER: aws-viewer-for-cbmc-release-ci
+  RELEASE_TAG: ${GITHUB_REF/refs\/tags\/}
+  VERSION: $(echo ${{ github.ref }} | cut -d "/" -f 3 | cut -d "-" -f 2)
```

Comment:
This could be just
```
$(echo $RELEASE_TAG | cut -d "-" -f 2)
```

Reply:
The above code will not work as `$RELEASE_TAG` will be replaced by `${GITHUB_REF/refs\/tags\/}` as string.
```
$(echo `${GITHUB_REF/refs\/tags\/}` | cut -d "-" -f 2) = '${GITHUB_REF/refs\/tags\/}'
```